lib/dbTasks.js

```diff
@@ -25,11 +25,12 @@ module.exports = {
 			};
 
 		buckets.forEach(function (table) {
-			var name = prefix + table;
+			var name = prefix + options.buckets ? options.buckets[table] : table;
```

Comment:
This is equal  to 
`var name = (prefix + options.buckets) ? options.buckets[table] : table;`
Which is not really what you want here.